AngularWebApi/AngularWebApi.csproj

```diff
@@ -130,6 +130,8 @@
     <Content Include="Scripts\modernizr-2.6.2.js" />
     <Content Include="Scripts\showErrors.js" />
     <Content Include="Scripts\showErrors.min.js" />
+    <Content Include="Scripts\underscore.js" />
```

Comment:
underscore was already part of the Mapping Edu application and I felt like
it has some useful functions.

On Sat, Jul 4, 2015 at 1:06 PM, Douglas Loyo <notifications@github.com>
wrote:

> In AngularWebApi/AngularWebApi.csproj
> <https://github.com/dougloyo/AngularWebApi2/pull/1#discussion_r33889113>:
>
> > @@ -130,6 +130,8 @@
> >      <Content Include="Scripts\modernizr-2.6.2.js" />
> >      <Content Include="Scripts\showErrors.js" />
> >      <Content Include="Scripts\showErrors.min.js" />
> > +    <Content Include="Scripts\underscore.js" />
>
> Why underscore?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dougloyo/AngularWebApi2/pull/1/files#r33889113>.
>